templates/glean_clients_daily_v1.sql

```diff
@@ -0,0 +1,104 @@
+WITH
+  unioned AS (
+  SELECT
+    submission_timestamp,
+    document_id,
+    client_info,
+    sample_id,
+    metadata,
+    normalized_channel,
+    metrics AS baseline_metrics,
+    NULL AS metrics
+  FROM
+    `moz-fx-data-shared-prod.org_mozilla_fenix.baseline_v1`
+  UNION ALL
+  SELECT
+    submission_timestamp,
+    document_id,
+    client_info,
+    sample_id,
+    metadata,
+    normalized_channel,
+    NULL AS baseline_metrics,
+    metrics
+  FROM
+    `moz-fx-data-shared-prod.org_mozilla_fenix.metrics_v1`
+  ),
+  --
+  base AS (
+  SELECT
+    DATE(submission_timestamp) AS submission_date,
+    LOWER(client_info.client_id) AS client_id,
+    *
+  FROM
+    unioned
+  WHERE
+    client_info.client_id IS NOT NULL ),
+  --
+  numbered_duplicates AS (
+  SELECT
+    *,
+    ROW_NUMBER() OVER (PARTITION BY client_id, submission_date, document_id ORDER BY submission_timestamp) AS _n
+  FROM
+    base ),
+  -- Deduplicating on document_id is necessary to get valid SUM values.
+  deduplicated AS (
+  SELECT
+    * EXCEPT (_n)
+  FROM
+    numbered_duplicates
+  WHERE
+    _n = 1 ),
+  windowed AS (
+  SELECT
+    submission_date,
+    client_id,
+    sample_id,
+    ROW_NUMBER() OVER w1_unframed AS _n,
+    'Fenix' AS app_name,
+    --
+    -- Take the earliest first_run_date if ambiguous.
+    MIN(SAFE.PARSE_DATE('%F', SUBSTR(client_info.first_run_date, 1, 10))) OVER w1 AS first_run_date,
+    --
+    -- Sums over distinct baseline pings.
+    SUM(udf_glean_timespan_seconds(baseline_metrics.timespan.glean_baseline_duration)) OVER w1 AS durations,
```

Comment:
Added a sum over `baseline_metrics.timespan.glean_baseline_duration`, assuming this is roughly equivalent to core ping `durations`.